assets/js/components/user-input/UserInputPreview.js

```diff
@@ -85,12 +84,12 @@ export default function UserInputPreview( props ) {
 								<UserInputPreviewGroup
 									questionNumber={ 1 }
 									title={ __(
-										'Which best describes your team/role relation to this site?',
+										'What are the goals of this site?',
```

Comment:
We need to update the copy for all three questions based on [this Figma design](https://www.figma.com/file/CdtrHB0HAjfGRWTnTMMQI0/SummaryWidget?node-id=917%3A24281).